UN-OCHA/common_design
# CD-502 Replace Twitter with X icon

## Types of changes

- :heavy_check_mark: Improvement (non-breaking change which iterates on an existing feature)

## Description
Replace Twitter logo with X
Screenshots on CD-502

##  Steps to test

  1. Check out this branch and clear cache
  2. Visit https://commondesign.test/demo and confirm the social media icons in the footer now display the X icon and that the hover state remains consistent with the other social media icons
  3. Visit https://commondesign.test/demo#cd-social-links and confirm the X icon appears there too.
  4. Also notice the file sizes are reasonable and are in the range of the other social media icons.

## Impact
I replaced the Twitter icon with the X icon keeping the same file names for ease. Many sub themes have their own SVG sprite in order to add their own icons (see https://github.com/UN-OCHA/unocha-site/blob/develop/html/themes/custom/common_design_subtheme/templates/layout/html.html.twig#L4) and https://github.com/UN-OCHA/unocha-site/blob/develop/html/themes/custom/common_design_subtheme/templates/cd/cd-icons/cd-icons.html.twig
The new X icons will need to be copied from the base theme to the sub theme and the `npm run svg:sprite` script run to regenerate the sub theme SVG sprite.


## PR Checklist

- [x] I have followed the Conventional Commits guidelines.
- [ ] I have made changes to the sub theme to reflect those in the base theme
